lib/CMakeLists.txt

```diff
@@ -585,13 +581,20 @@ add_custom_command(
   PRE_LINK
   COMMAND ${CMAKE_COMMAND} -E remove ${CMAKE_CURRENT_BINARY_DIR}/libquda.a ${CMAKE_CURRENT_BINARY_DIR}/libquda.so)
 
+set_target_properties(quda PROPERTIES INSTALL_RPATH_USE_LINK_PATH ON )
+
 install(
   TARGETS quda
-  EXPORT qudaTargets
+  EXPORT QUDATargets
```

Comment:
Went back to use same captialization as package name

Reply:
I just rebased this with current `develop` to make it up-to-date with that. Includes @weinbe2 's most recent patch